release.json

```diff
@@ -1,5 +1,6 @@
 {
-  "mongodb-kubernetes-operator": "0.7.5",
+  "golang-builder-image": "golang:1.18.5",
+  "mongodb-kubernetes-operator": "0.0.1",
```

Comment:
Why this is `0.0.1`? I suspect this is either a leftover or you wanted to leave it like this for testing. If that is so, you might also use `0.7.5+<SHA from git>`. In semantic versioning everything after the plus sign is treated as build id. Alternatively, if you're using this for testing, you might want to use the minus sign (so that this will be treated as pre-release and won't replace the original release).

Reply:
Yes, this shouldn't be here. Thanks for the idea with SHA!